evap/staff/tests/test_tools.py

```diff
@@ -9,6 +10,7 @@
 from model_bakery import baker
 from openpyxl import load_workbook
 
+from evap import settings
```

Comment:
Django has this "weirdness" that you [shouldn't import your settings module directly](https://docs.djangoproject.com/en/5.1/topics/settings/#using-settings-in-python-code), and instead always just use `from django.conf import settings`. It should be a drop-in replacement here.

```suggestion
from django.conf import settings
```